relatio/_embeddings.py

```diff
@@ -239,3 +262,44 @@ def _load_keyed_vectors(self, model):
             raise
 
         return api.load(model)
+
+    def _get_default_vector(self, phrase: str) -> np.ndarray:
+
+        tokens = phrase.split()
+        embeddable_tokens = []
+        for token in tokens:
+            if token in self._vocab:
+                embeddable_tokens.append(token)
+            else:
+                warnings.warn(
+                    f"No vector for token: {token}. It is not used to compute the embedding of: {phrase}.",
+                    RuntimeWarning,
+                )
+        res = np.mean([self._model[token] for token in embeddable_tokens], axis=0)
+        return res
+
+
+def _compute_distances(vectors1, vectors2):
+    distances = cdist(vectors1, vectors2, metric="euclidean")
+    return distances
+
+
+def _get_min_distances(distances):
+    return np.min(distances, axis=1)
+
+
+def _get_index_min_distances(distances):
+    return np.argmin(distances, axis=1)
```

Comment:
I am not 100% sure whether we need them. They are part of standard libraries.

Reply:
I added them mainly for programmers reading the NarrativeModel() code. I find it easier to read with clear function names (more like natural language). What do you think?

Reply:
I agree. We should make them public and write the documentation.

Reply:
I wrote the basic documentation.

Reply:
I will keep like this.